flytekit/__init__.py

```diff
@@ -1,4 +1,4 @@
 from __future__ import absolute_import
 import flytekit.plugins
 
-__version__ = '0.1.5'
+__version__ = '0.1.6'
```

Comment:
you probably are doing this, but  you can submit this after your other PR so you get both changes the version bump

Reply:
Yeah in the future I will bump the version via an explicit PR.